build.gradle

```diff
@@ -193,6 +198,17 @@ task fatJar(type: Jar) {
     with jar
 }
 
+shadowJar {
```

Comment:
great, could you delete fatJar as part of this then?  No need to have both if the shadow jar includes it all.